lambdas/readIncidentHandler/index.js

```diff
@@ -0,0 +1,54 @@
+require('aws-sdk');
+
+exports.handler = async (event) => {
+    const dataClient = require('data-api-client')({
+        secretArn: process.env.SECRET_ARN,
+        resourceArn: process.env.RDS_DB_ARN,
+        database: 'postgres'
+    });
+
+    const filter = event.queryStringParameters;
+    return {
+        statusCode: 200,
+        body: mapOutput(await selectIncidents(dataClient, filter))
```

Comment:
❓ If an error throw here, you don't want to catch and output some custom message and preffer to let AWS Lambda handle it ?